lib/internal/safe_globals.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const copyProps = (unsafe, safe) => {
+  for (const key of [...Object.getOwnPropertyNames(unsafe),
+                     ...Object.getOwnPropertySymbols(unsafe)
+  ]) {
+    if (!Object.getOwnPropertyDescriptor(safe, key)) {
+      Object.defineProperty(
+        safe,
+        key,
+        Object.getOwnPropertyDescriptor(unsafe, key));
+    }
+  }
+};
+const makeSafe = (unsafe, safe) => {
```

Comment:
We haven't had much of a defined policy about this. There have been cases where we have avoided changing *existing* exports from regular functions to arrow functions but given that this is a new API, I'm not sure if there's an issue here.